### PR TITLE
Added SR5, SR10 and SR13

### PR DESCRIPTION
**Pull request by upasanapattnaik-eq**
Added queries and README for:
+ SR5-Cases_of_Reportable_Diseases_by_State 
+ SR10-Multi_Year_Line_Graph_of_Disease_Cases
+ SR13-Counts_of_Selected_Diseases_By_Case_Status

### CODE REVIEW

**Review by rseshadri99 — APPROVED** [review]
Looks good. One generic comment is we need to identify how to display dynamic column names for Month/Year based report table/columns

File: source/analytics/reports/jurisdictions/cdc-central/report-sql/SR10-Multi_Year_Line_Graph_of_Disease_Cases/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# SR10: Multi-Year Line Graph of Disease Cases
+
+## Introduction
+
+Standard report SR10 or Multi-Year Line Graph of Disease Cases generates a multi-year line graph of a disease given its selected state and timeframe.
```
Comment (alog-enquizit) [inline source/analytics/reports/jurisdictions/cdc-central/report-sql/SR10-Multi_Year_Line_Graph_of_Disease_Cases/README.md:5]: nitpick (optional): Consider adding a punctuation after the name of the report.
`Standard report SR10 or Multi-Year Line Graph of Disease Cases : This report generates...`

**Review by alog-enquizit — APPROVED** [review]
LGTM 👍

File: source/analytics/reports/jurisdictions/cdc-central/report-sql/SR10-Multi_Year_Line_Graph_of_Disease_Cases/README.md (hunk 1 above)
Comment (upasanapattnaik-eq) [reply source/analytics/reports/jurisdictions/cdc-central/report-sql/SR10-Multi_Year_Line_Graph_of_Disease_Cases/README.md:5]: No no it is appreciated! Let me update the content accordingly.

**Comment by alog-enquizit** [comment]
@upasanapattnaik-eq Thanks for making the changes. Looks good for merging!

**Pull request merged**